provision/acc_provision/apic_provision.py

```diff
@@ -228,6 +240,32 @@ def get_phys_dom(self, domain):
         path = "/api/mo/uni/phys-%s.json" % domain
         return self.get_path(path)
 
+    def check_aep_association_to_phys_dom(self, phy_dom, aep):
```

Comment:
This is something we can associate.. instead of checking and returning error

Reply:
Sure, I have updated.